src/controllers/main/main.ts

```diff
@@ -2349,14 +2357,35 @@ export class MainController extends EventEmitter {
 
       // broadcast through bundler's service
       let userOperationHash
+      const bundler = bundlerSwitcher.getBundler()
       try {
-        userOperationHash = await bundler.broadcast(userOperation, network!)
+        userOperationHash = await bundler.broadcast(userOperation, network)
       } catch (e: any) {
+        let retryMsg
+
+        // if the signAccountOp is still active (it should be)
+        // try to switch the bundler and ask the user to try again
+        // TODO: explore more error case where we switch the bundler
+        if (this.signAccountOp) {
+          const decodedError = bundler.decodeBundlerError(e)
+          const humanReadable = getHumanReadableBroadcastError(decodedError)
```

Comment:
I'm pretty sure that reason from `decodedError` is enough and `getHumanReadableBroadcastError` isn't needed